src/components/metadataEditor/metadataEditor.js

```diff
@@ -212,19 +233,19 @@ define(['itemHelper', 'dom', 'layoutManager', 'dialogHelper', 'datetime', 'loadi
     function addElementToList(source, sortCallback) {
         require(['prompt'], function (prompt) {
```

Comment:
```suggestion
        import('prompt').then(({default: prompt}) => {
```

Reply:
This requires to change `prompt.default` down below.